src/components/legend/color.tsx

```diff
@@ -49,16 +49,17 @@ export const Color = memo(
     });
 
     return (
-      <div
+      <button
+        type="button"
```

Comment:
Also this should have a `tabIndex={0}` when `hasColorPicker` is false

```suggestion
        type="button"
        tabIndex={hasColorPicker ? undefined : 0}
        
```

Reply:
> Also this should have a `tabIndex={0}` when `hasColorPicker` is false

Why's that? Won't it not have any action when `hasColorPicker == false`? If so, then it shouldn't be a `<button>` and it shouldn't be a tab stop (i.e., `tabIndex={0}`). If it does still have an action, then it should stay a `<button>` which doesn't need a `tabIndex` defined.

Reply:
I think f7e2f27 is working as expected with your feedback, let me know what you think (and if I'm misunderstanding) when you have a chance. Thank you!